### DISCUSSION TITLE
[2022/01/25] Frontend Night

### DISCUSSION THREAD

**Discussion by jjenko** [wantedly/frontend-night#189] [Archive]
## Policy

- 資料の準備をがんばらない
- 参加・途中退席は自由
- スキップしない

話したい・聞きたいネタを書いてく ✏️

ハッシュタグ [#frontend_night](https://twitter.com/hashtag/frontend_night?src=hash)

Prev: https://github.com/wantedly/frontend-night/discussions/188 

 _Created from https://github.com/wantedly/frontend-night/discussions/179 by [issue-creator](https://github.com/rerost/issue-creator)_

**Comment by kobayang** [comment]
[エンジニアのための Figma 知識](https://zenn.dev/seya/articles/127027b75dbba0)

seyaさんの記事。Figma使う上で必須機能だと思った（特に小ネタ）

- 小ネタ: Cmd を押しながらクリックで最下層の要素を選択できる
- 小ネタ2: レイヤー間を移動するショートカット

**Reply by kobayang** [reply]
![スクリーンショット 2022-01-25 18 39 55](https://user-images.githubusercontent.com/12181179/150951507-63328b32-f16d-4d0c-857f-6c228158abeb.png)

command + y でワイヤーでデザインが見れる
構造が知れて便利

**Reply by kobayang** [reply]
![スクリーンショット 2022-01-25 18 43 38](https://user-images.githubusercontent.com/12181179/150952140-d34a0aa0-d08b-4498-bc9f-eaf50172c850.png)

Figma の frame 単位で Link を取得できる

**Comment by kobayang** [comment]
使ってないけど、便利そうだな〜と思った。

https://forest.watch.impress.co.jp/docs/news/1382919.html

**Reply by kobayang** [reply]
alfred で全部できる説。ワークフローを ruby で書けるのでそれでなんでもできる。
